text/052-platform-release-cycle.md

```diff
@@ -0,0 +1,137 @@
+built. Coq point release are then used as a *quick* mean to deliver fixes
+to the platform.
+
+# Motivation
+
+The beta testing period does not provide enough feedback to sensibly improve
+Coq between the `VX+beta1` and `VX.0` tags. The reasons are, most important
+last:
+- users have better things to do, frankly
+- one month is short, especially with Christmas/summer holydays around
+- in order to test Coq you are likely to need a bunch of Coq packages, and they
+  are typically only available after the beta period is over
+
+Moreover the "beta period" is also seen by devs as a time frame where they can
+still change many things. As a results users are even less happy to spend time
+working on a moving target.
+
+Finally, due to the cost of backporting breaking changes from the master
+branch to the release branch, during 8.12 and 8.13 release cycle this only
+happened very early, just after branching (only 2 in the beta period for 8.12,
+0 for 8.13).
+
+## New actor: platform
+
+The Coq platform is a selection of Coq package versions which work well
+for a given Coq version. One of its objective is to disentangle the release
+of Coq with the release of other packages. It provides scripts that build
+that selection of Coq packages on Windows, Mac and Linux from sources
+on the user's machine, and as well binary installers (prebuilt) for users
+to quickly get a working environment.
+
+This product is what users will test in this new model; Coq is a core
+component of it.
+
+# Detailed design
+
+The new process' timeline:
+```
+                     D             D            D   D  D
+coq      --+---(1)---+------(2)----+-----(3)----+---+--+----
+ vX branch/          |             |            |
+            X+rc tag/              |            |
+                           X.0 tag/             |
+                                        X.1 tag/
+
+                                   I                    I
+platform  -----------+----(4)------+--------(5)---------+
+           vX branch/              |                    |
+                        X+beta tag/                     |
+                                                X.0 tag/
+Artifacts:
+- D = docker image for Coq (and opam package)
+- I = binary installers for Coq platform (and opam packages and docker image)
+```
+
+## Coq
+
+On time based schedule the RM branches vX.
+
+1. The RM shepherds the few PR which are ready and pins projects tracked by   CI. Then he tags. This should take 10 days. No packages are built, just a git
+tag. A docker image is build, to that upstreams can use them in CI. No breaking change is allowed from now on.
+2. Doc is updated (eg. Changes file) and eventual fixes required by the platform
+are done. Ideally no other change is done. This should take 10 days.
+3. In response to a severe bug report Coq devs make an hotfix and the RM tags a point release. Once the fix is produced, a tag comes in 2 days.
```

Comment:
Shouldn't the commitment be with respect to the moment the fix is merged in `master`?